parallel/client.py

```diff
@@ -192,11 +192,16 @@ def verify(self):
         """
         See abstract class for documentation.
         """
-        futures = [stub.Status.future(service_pb2.StatusRequest()) for stub in self.stubs]
-        workers_status = [x.result() for x in futures]
```

Comment:
Previously an exception could be raised here that said "not all workers could be connected to", which made it difficult to debug if an instance was acting up. This will have the ip address in the exception allowing for easier debugging of borked instances

Reply:
Nice! this was intentionally designed this way so you can replace the list comprehension with something more useful later :)